expr/helper/helper.go

```diff
@@ -161,7 +143,7 @@ func ForEachSeriesDo(ctx context.Context, e parser.Expr, from, until int64, valu
 	var results []*types.MetricData
 
 	for _, a := range arg {
-		r := a.CopyTag(e.Target()+"("+a.Name+")", a.Tags)
+		r := a.CopyName(e.Target() + "(" + a.Name + ")")
```

Comment:
CopyName do copy of tags. We really need them ? Where tags are ovewtitten ?

Reply:
There are the functions that call `ForEachSeriesDo`:
- `absolute`
- `derivative`
- `integral`
- `invert`
- `isNotNull`
- `offsetToZero`

And they all seem to modify the tags. I see your point though: a function calling `ForEachSeriesDo` might not necessarily want to change the tags, so this mightbe unnecessary overhead for a future use case. Maybe we could have the functions I enumerated above copy the tags manually as they need, but I think this is not necessary now. Let me know what you think,

Reply:
Yes, you are right. May be rename ForEachSeriesDo and ForEachSeriesDo1 with more adequate name. But not in this PR.